.teamcity/components/generated/services.kt

```diff
@@ -30,6 +30,7 @@ var services = mapOf(
         "eventgrid" to "EventGrid",
         "eventhub" to "EventHub",
         "frontdoor" to "FrontDoor",
+        "hardwaresecuritymodule" to "Hardware Security Module",
```

Comment:
this is a bit long, could we maybe abbreviate it to 
```suggestion
        "hsm" to "Hardware Security Module",
```

Reply:
updated